src/main/resources/compendium.yaml

```diff
@@ -205,13 +205,18 @@ components:
           description: the library artifact version
     IdlName:
       type: string
-      description: the names of the set of protocols supported in compendium
+      description: the names of the set of protocols supported to upload to compendium
       enum:
         - avro
         - protobuf
         - mu
         - openapi
         - scala
+    IdlNameTarget:
+      type: string
+      description: the names of the set of protocols supported in compendium to be target
+      enum:
+        - mu
```

Comment:
Do we only support `mu` :thinking: ?

Reply:
As a target protocol, yes. @Yawolf said it before
> Nope, We can store any of the IDLs, but we can only translate:
> Avro -> Mu
> Protobuf -> Mu